filters/shedder/admission.go

```diff
@@ -370,6 +370,13 @@ func sum(a []int64) int64 {
 	return result
 }
 
+func (ac *admissionControl) setCommonTags(span opentracing.Span) {
+	span.SetTag("admissionControl.group", ac.metricSuffix)
+	span.SetTag("admissionControl.mode", ac.mode.String())
+	span.SetTag("admissionControl.d", ac.d.String())
```

Comment:
what is d? I think the tag in the span should be a bit descriptive.

Reply:
It's actually the duration during which admissionControll count requests amount. I have renamed "d" to "duration".